GitHub Desktop/GitHub Desktop.jss.recipe

```diff
@@ -17,7 +17,7 @@
 		<key>NAME</key>
 		<string>GitHub Desktop</string>
 		<key>OS_REQUIREMENTS</key>
-		<string>10.15.x,10.14.x,10.13.x,10.12.x,10.11.x,10.10.x</string>
+		<string>10.16.x,10.15.x,10.14.x,10.13.x,10.12.x,10.11.x,10.10.x</string>
```

Comment:
I haven't tested Jamf's package requirements with Big Sur yet — is `11.x` also valid?

Reply:
Tested on 10.25.1 and adding `10.16.x` to `OS_REQUIREMENTS` works successfully. I expect there will be further change to Jamf's OS reporting that will make adding `11.x` necessary in the future but not at present. Not sure that having `OS_REQUIREMENTS` set in the recipes is the right thing to do as the logic should probably be in the scoping but that may be another discussion.

Reply:
@d33t5 - Could you verify that `10.16.x` is still working for you? Another contributor mentioned that only `11.x` allows Big Sur Macs to install packages (https://github.com/autopkg/jss-recipes/pull/235#issuecomment-740005770).